Automattic/jetpack
# Boost: Refactor Image CDN features into sub modules

This PR started out as me wanting to get Liar and Quality Settings to show in the JP debugger. However, the need for sub modules happened organically as it makes the most sense to me. To avoid doing too many things into one PR, I decided to leave the state to another PR.

## Proposed changes:

* Introduce sub modules;
* Move Image CDN features (Liar and Quality Settings) to sub modules.

### Other information:


## Jetpack product discussion



n/a

## Does this pull request change what data or activity we track or use?



no

## Testing instructions:






* Setup Boost;
* Make sure modules work as expected.
